compiler/nimeval.nim

```diff
@@ -125,3 +125,12 @@ proc createInterpreter*(scriptName: string;
 proc destroyInterpreter*(i: Interpreter) =
   ## destructor.
   discard "currently nothing to do."
+
+proc runRepl*(r: TLLRepl;
+              searchPaths: openArray[string] = [findNimStdLibCompileTime()];
+              flags: TSandboxFlags = {}) =
```

Comment:
I think the TSandboxFlags should be more restrictive, good defaults and all that.